packages/Spellcheck-Core.package/SpellingTextParser.class/instance/parseCamelCaseText..st

```diff
@@ -0,0 +1,19 @@
+as yet unclassified
+parseCamelCaseText: aString
```

Comment:
If I see it right, this method only parses camelcase-Words and not whole Texts. I think it would be a good idea to rename the method to `parseCamelCaseWord`. This would make it easier to understand that `parseCamelCaseWord` is called after `parseText`.

Reply:
Yes, you are right about that.